src/frontend/public/locales/common/en-US.json

```
@@ -469,6 +499,9 @@
   "Use SSL": "Use SSL",
   "Value": "Value",
   "Variables": "Variables",
+  "Weekly": "Weekly",
+  "While the signature is disabled, it will not be available to the users.": "While the signature is disabled, it will not be available to the users.",
```

Biome (2.1.2)

[error] The key While the signature is disabled, it will not be available to the users. was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)